packages/acceptit-Tests.package/ACLibraryParserTest.class/instance/setUp.st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+setUp
+	parser := ACLibraryParser new.
```

Comment:
Why do you initialize a `parser` instance variable but still use `ACLibraryParser new` in `testItParsesAStep` and `testItParsesSomethingToAMehtod`?

Reply:
just checking if you actually do reviews :P